tests/integration/api/test_api.py

```diff
@@ -959,3 +961,9 @@ def test_roles_crud(self, dog):
         # check if new role is deleted successfully
         res = dog.Roles.get(role_uuid)
         assert "errors" in res
+
+    @mock.patch('datadog.api._return_raw_response', True)
+    def test_user_agent(self, dog):
+        _, resp = dog.api_client.APIClient.submit('GET', 'validate')
+        assert re.match(r'^datadogpy\/[^\s]+ \(python [^\s]+; os [^\s]+; arch [^\s]+\)$', resp.request.headers['User-Agent'])
+        resp.request.headers['User-Agent']
```

Comment:
Is this line needed?

Reply:
Ah, forgotten debugging line, sorry. Will remove.